src/Command/SynchronizeTaxonCommand.php

```diff
@@ -49,20 +49,22 @@ public function __construct(
         $this->taxonRepository = $taxonRepository;
     }
 
-    protected function configure()
+    protected function configure():void
     {
         $this->setName('sulu-sylius:synchronize:taxon')
             ->setDescription('Synchronize taxon tree to Sulu');
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
         // disable logger because of memory issues
         $this->entityManager->getConfiguration()->setSQLLogger(null);
         $this->entityManager->getConnection()->getConfiguration()->setSQLLogger(null);
         gc_enable();
 
         $this->syncTaxonTree($output);
+
+        return 1;
```

Comment:
this indicates that an error happened, right?

Reply:
🙈 your right